package/MDAnalysis/analysis/rms.py

```diff
@@ -235,13 +235,16 @@ def rmsd(a, b, weights=None, center=False, superposition=False):
 
     Example
     -------
-    >>> u = Universe(PSF,DCD)
+    >>> import MDAnalysis as mda
+    >>> from MDAnalysis.analysis.rms import rmsd
+    >>> from MDAnalysis.tests.datafiles import PSF, DCD
+    >>> u = mda.Universe(PSF, DCD)
     >>> bb = u.select_atoms('backbone')
     >>> A = bb.positions.copy()  # coordinates of first frame
-    >>> u.trajectory[-1]         # forward to last frame
```

Comment:
Don't assign to `last_frame` just go forwards in place but other than that looks great!

Reply:
you can do `_ = u.trajectory[-1]` I think thats a good compromise.

Reply:
Yes - I have updated the code.